### ISSUE TITLE
Cannot import module -- circular import error

### ISSUE THREAD

**Issue by dgervini** [ttm/music#13]
I'm getting this error when I try to import the package:

<img width="626" alt="Screenshot 2022-06-27 211359" src="https://user-images.githubusercontent.com/69220462/176226862-b6f65a5a-9cde-4091-bb79-68a4286ee7a5.png">

**Comment by kennyjacobson** [comment]
Same with me.

**Comment by ttm** [comment]
should be solved with version 0.7b3 (published today).

**Comment by ttm** [comment]
@dgervini @kennyjacobson — closing the loop properly, rather than leaving this on "should be solved with version 0.7b3".

The screenshot was the circular import: `music/__init__.py` imported `utils`, which imported `core`, which imported back into the partially-initialised `music` package. It fired at import time on a clean install, so `import music` was simply impossible — there was no usage error to find on your side, which is presumably why two of you hit exactly the same wall a month apart.

**Fixed, and now checked on every release.** `import music` from an empty virtualenv is part of the release procedure, and CI exercises the whole public API on Python 3.10 through 3.13 — 481 tests at 100% coverage, enforced.

```console
pip install -U music
```

```python
import music
music.write_wav_mono(music.note(440, 1), "a.wav")
```

Verified on 1.1.1 from a clean environment. There is a [tutorial](https://ttm.github.io/music/tutorial.html) now if you want to pick it up again.
